diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index 8fb94c5..b333663 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -1,8 +1,6 @@
 import React from 'react';
 
-import { Box, Button, Flex, Typography } from '@strapi/design-system';
-import { Link } from '@strapi/design-system/v2';
-import { ArrowRight } from '@strapi/icons';
+import { Box, Flex, Typography } from '@strapi/design-system';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 import styled from 'styled-components';
@@ -30,33 +28,8 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
                 })}
           </Typography>
           <WordWrap textColor="neutral600" variant="epsilon">
-            {hasCreatedContentType
-              ? formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content.again',
-                  defaultMessage:
-                    'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
-                })
-              : formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content',
-                  defaultMessage:
-                    'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
-                })}
+            Welcome to Next Games Admin! Add more games, categories and publishers from Content Manager.
           </WordWrap>
-          {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
-              })}
-            </Link>
-          ) : (
-            <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
-              {formatMessage({
-                id: 'app.components.HomePage.create',
-                defaultMessage: 'Create your first Content type',
-              })}
-            </Button>
-          )}
         </Flex>
       </Box>
     </div>
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index 21ab701..fac7873 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -79,10 +79,7 @@ export const HomePageCE = () => {
             </GridItem>
           </Grid>
           <Grid gap={6}>
-            <GridItem col={8} s={12}>
-              {showGuidedTour ? <GuidedTourHomepage /> : <ContentBlocks />}
-            </GridItem>
-            <GridItem col={4} s={12}>
+            <GridItem col={12} s={12}>
               <SocialLinks />
             </GridItem>
           </Grid>
